sleap_nn/training/model_trainer.py

```
@@ -487,10 +493,11 @@ def run_subprocess():
                 run_subprocess()
 
             except Exception as e:
-                raise Exception(f"Error while creating the `.bin` files... {e}")
+                logger.error(f"Error while creating the `.bin` files... {e}")
+                raise Exception()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)